scripts/releaseHotels.js

```diff
@@ -2,13 +2,26 @@
 
 const path = require('path');
 const child_process = require('child_process');
+const fs = require('fs');
 
 const exec = (command, options) =>
   child_process.execSync(command, {
     stdio: 'inherit',
     ...options,
   });
 
+const getFileContentAsObject = (filePath, options) => {
```

Comment:
I will show you a magic here:

```
▲ react-native-app at master ✔ node                                    
> const packageInfo = require('./app/hotels/package.json')
undefined
> console.log(packageInfo.version)
0.0.0
undefined
> 
```

This is what I meant with:

> Because hey! We are in JS! 🙂